cli/main.py

```diff
@@ -1,28 +1,50 @@
-from helper import olc_to_phrase, two_to_word
-
 import typer
 
+import helper
+
 
 app = typer.Typer()
 
 
 @app.command()
 def get_word(char_code: str):
-    word = two_to_word(char_code)
+    word = helper.get_word(char_code)
     typer.echo(f"{word}")
 
 
 @app.command()
-def get_phrase(code: str):
+def get_code(word: str):
+    code = helper.get_code(word)
+    typer.echo(f"{code}")
+
+
+@app.command()
+def get_phrase(olc: str, single: bool = False):
     try:
-        if len(code) != 12:
```

Comment:
I think this should be `if len(code) > 12:` because 6PH57VP3+PR6 is a valid code, but 6PH5 is also a valid code, same with 6P. The more characters you have just narrows down your position, but AFAIK [codes can be shorter than 12](https://en.wikipedia.org/wiki/Open_Location_Code)?

Reply:
yeah, according to google maps ones in my area are 6 char

Reply:
how are you able to see your OLC code in Google Maps (website or mobile app?) i can't seem to find mine unless i'm just blind

Reply:
https://plus.codes

Reply:
![image](https://user-images.githubusercontent.com/43295248/83685161-303a9d00-a5b6-11ea-8eb9-9763aab44fb4.png)
also on my phone